ChangeLog

```diff
@@ -1,6 +1,12 @@
+<<<<<<< HEAD
```

Comment:
These weird looking lines are left overs from when you merged two branches that contained conflicting changes. In this case the thing to do is to delete them.